pkg/util/ring_config.go

```diff
@@ -82,6 +82,27 @@ func (cfg *RingConfig) ToLifecyclerConfig(numTokens int) (ring.BasicLifecyclerCo
 	}, nil
 }
 
+func CortexLifecyclerConfigToRingConfig(cfg ring.LifecyclerConfig) RingConfig {
```

Comment:
I think this should be dskit instead of cortex?

Reply:
I think we will do that eventually, but we've not yet migrated to the `dskit` ring.